package.json

```
@@ -13,8 +13,10 @@
     "lint": "eslint '{src,test}/**/*.ts' --fix",
     "format": "prettier --write '{src,test}/**/*.ts'",
     "scrape": "tsx prisma/scraper.ts",
-    "scrape:scheduled": "SCHEDULED_MODE=true node dist/prisma/scraper.js",
-    "studio": "prisma studio"
+    "scrape:scheduled": "SCHEDULED_MODE=true tsx prisma/scraper.ts",
```

```suggestion
    "scrape:scheduled": "SCHEDULED_MODE=true node dist/prisma/scraper.js",
```
This was changed on main so that once the app is built, the dev and prod servers could run it directly using node instead of needing a typescript engine